app/editor/src/features/content/tool-bar/ContentToolBar.tsx

```diff
@@ -46,40 +46,16 @@ export const ContentToolBar: React.FC<IContentToolBarProps> = ({ onSearch }) =>
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const onFilterChange = (filter: IContentListFilter) => {
-    storeFilter(filter);
-    replaceQueryParams({ ...filter, ...filterAdvanced }, { includeEmpty: false });
-  };
-
-  // pass filter and filter advanced so we don't lose the advanced filter when we change the filter
-  const onAdvancedFilterChange = (advancedFilter: IContentListAdvancedFilter) => {
-    storeFilterAdvanced(advancedFilter);
-    replaceQueryParams(
-      { ...filter, ...advancedFilter },
-      {
-        includeEmpty: false,
-        convertObject: (value) => {
-          if (instanceOfIOption(value)) return value.value;
-          return value.toString();
-        },
-      },
-    );
-  };
-
   return (
     <ToolBar>
       {/* first section */}
       <CreateNewSection />
       {/* second section */}
-      <FilterContentSection
-        onChange={onFilterChange}
-        onAdvancedFilterChange={onAdvancedFilterChange}
-        onSearch={onSearch}
-      />
+      <FilterContentSection />
```

Comment:
All of these components use the redux state, there is no point in passing down state through properties since they are already tightly bound to redux state.